examples/hub-spoke/variables.tf

```diff
@@ -0,0 +1,157 @@
+}
+
+variable "service_gateway_enabled" {
+  description = "whether to create a service gateway"
+  type        = bool
+  default     = false
+}
+
+variable "tags" {
+  description = "simple key-value pairs to tag the resources created"
+  type        = map(any)
+  default = {
+    terraformed = "yes"
+    module      = "oracle-terraform-modules/vcn/oci"
+  }
+}
+
+variable "vcn_cidr" {
+  description = "cidr block of VCN"
+  type        = string
+  default     = "10.0.0.0/16"
+}
+
+variable "vcn_dns_label" {
+  description = "A DNS label for the VCN, used in conjunction with the VNIC's hostname and subnet's DNS label to form a fully qualified domain name (FQDN) for each VNIC within this subnet"
+  type        = string
+  default     = "vcnmodule"
+}
+
+variable "vcn_name" {
+  description = "user-friendly name of to use for the vcn to be appended to the label_prefix"
+  type        = string
+  default     = "vcn-module"
+}
+
+# gateways parameters
+
+variable "drg_display_name" {
+  description = "(Updatable) Name of Dynamic Routing Gateway. Does not have to be unique."
+  type        = string
+  default     = "drg"
+}
+
+# routing rules
+
+variable "internet_gateway_route_rules" {
+  description = "(Updatable) List of routing rules to add to Internet Gateway Route Table"
+  type = list(object({
+    destination       = string
+    destination_type  = string
+    network_entity_id = string
+    description       = string
+  }))
+  default = null
+}
+
+# locals {
+#   nat_gateway_route_rules = [ # this is a local that can be used to pass routing information to vcn module for either route tables
+#     {
+#       destination       = "192.168.0.0/16" # Route Rule Destination CIDR
+#       destination_type  = "CIDR_BLOCK"     # only CIDR_BLOCK is supported at the moment
+#       network_entity_id = "drg"            # for nat_gateway_route_rules input variable, you can use special strings "drg", "nat_gateway" or pass a valid OCID using string or any Named Values
+#       description       = "Terraformed - User added Routing Rule: To drg created by this module. drg_id is automatically retrieved with keyword drg"
+#     },
+#     {
+#       destination       = "172.16.0.0/16"
+#       destination_type  = "CIDR_BLOCK"
+#       network_entity_id = module.vcn.drg_id
+#       description       = "Terraformed - User added Routing Rule: To drg with drg_id directly passed by user. Useful for gateways created outside of vcn module"
+#     },
+#     {
+#       destination       = "203.0.113.0/24" # rfc5737 (TEST-NET-3)
+#       destination_type  = "CIDR_BLOCK"
+#       network_entity_id = "nat_gateway"
+#       description       = "Terraformed - User added Routing Rule: To NAT Gateway created by this module. nat_gateway_id is automatically retrieved with keyword nat_gateway"
+#     },
+#     {
+#       destination       = "192.168.1.0/24"
+#       destination_type  = "CIDR_BLOCK"
+#       network_entity_id = oci_core_local_peering_gateway.lpg.id
```

Comment:
This additional lpg example needs to be the in root terraform.tfvars.example too.